migrator/migrations/m_111_to_m_112_groups_invalid_values/binenc/byte_slice_list.go

```diff
@@ -0,0 +1,55 @@
+package binenc
+
+import (
+	"bytes"
+	"encoding/binary"
+	"io"
+
+	"github.com/pkg/errors"
+)
+
+// WriteBytesList takes a list of byte slices and writes them in encoded form to a writer.
+func WriteBytesList(w io.Writer, byteSlices ...[]byte) (int, error) {
+	var total int
+	for _, byteSlice := range byteSlices {
+		n, err := WriteUVarInt(w, uint64(len(byteSlice)))
+		total += n
+		if err != nil {
+			return total, err
+		}
+
+		n, err = w.Write(byteSlice)
+		total += n
+		if err != nil {
+			return total, err
+		}
+	}
+	return total, nil
+}
+
+// EncodeBytesList takes a list of byte slices and encodes them into a single byte slice.
+func EncodeBytesList(byteSlices ...[]byte) []byte {
```

Comment:
Who is using it?

Reply:
I followed the same examples for copying stackrox code into the migrations - which is copying the package as is and not modifying it, even though some functions may be unused.